read-cache.c

```diff
@@ -1669,9 +1669,6 @@ static int verify_hdr(struct cache_header *hdr, unsigned long size)
 	if (!verify_index_checksum)
 		return 0;
 
-	if (gvfs_config_is_set(GVFS_SKIP_SHA_ON_INDEX))
```

Comment:
Can we remove the other places this bit is used and in the `gvfs.h` file and the `config.txt` file?

Reply:
We can't remove it entirely because we still have the GVFS specific behavior to not update the sha on write which uses the same flag.  That part never made it upstream and I don't believe it ever will.

I will update config.txt to reflect that this flag is only used on write.